docs/AboutUs.adoc

```diff
@@ -3,53 +3,47 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
-_{The dummy content given below serves as a placeholder to be used by future forks of the project.}_ +
+NUSCouples was developed by team https://github.com/CS2103JAN2018-T09-B4/main[T09-B4]. +
+ +
 {empty} +
 We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
 
 == Project Team
 
-=== John Doe
-image::damithc.jpg[width="150", align="left"]
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+=== Samuel Gan
+image::samuel.jpeg[width="150", align="left"]
+{empty}[https://github.com/https://github.com/traceurgan[github]]
 
-Role: Project Advisor
+Role: Developer +
+Responsibilities: Model
 
 '''
 
-=== John Roe
-image::lejolly.jpg[width="150", align="left"]
-{empty}[http://github.com/lejolly[github]] [<<johndoe#, portfolio>>]
+=== Marlene Koh
+image::marlene.jpeg[width="150", align="left"]
+{empty}[https://github.com/marlenekoh[github]]
 
-Role: Team Lead +
+Role: Developer +
```

Comment:
I think someone need to be the team leader.

Reply:
I'll rise up and take the role of team leader if everyone else is agreeable.

For my leadership style, after much contemplation, I am deciding to go for the servitude leadership style. This means that as a leader, I do not only have to be COMPLETELY accountable to my entire team for my actions, to decide on a path to take etc etc..., BUT in the hierarchy structure of the group, the LEADER is at the BOTTOM while all the other MEMBERS are at the TOP above the LEADER.

Many organizations such the international Toastmasters organization also uses a committee structure whereby it is said members are higher in ranking than elected committee members!

I STRONGLY feel it would be a PHENOMENAL success if we not only MAKE IT as a team just before week 13's DEMO, but we COME OUT AT THE VERY TOP!! :+1: